db/models/donation.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Donation = db.define('donations', {
+  date: {
+    type: Sequelize.DATE,
+    defaultValue: Sequelize.NOW
+  },
+  amount: {
+    type: Sequelize.INTEGER, // in U.S. Cents
+    allowNull: false
+  },
+  method: {
+    type: Sequelize.ENUM('stripe', 'check', 'other')
+  },
+  thanked: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  },
+  dedication: {
+    type: Sequelize.STRING
```

Comment:
That 'thanked' flag was something Rachel wanted, I wasn't sure if she had thought through how she wanted this to work or not. I think the thank-you could really be given on the page following the donation, but I think she wanted to be more personal